### PR TITLE
Balance Air Units

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
This PR attemps to fix https://github.com/OpenHV/OpenHV/issues/882 and fix https://github.com/OpenHV/OpenHV/issues/906, rebalances Banshees and Speeders, rebalances air transports of both factions and rebalances anti-air turrets.

### CODE REVIEW

**Comment by Mailaender** [comment]
Needs a rebase. Also, can you summarize your changes for the changelog? Your commit messages are pretty bad.

**Comment by WilliamWeberBerrutti** [comment]
Maybe it should be better if I could fixup some commits? During the development, I was so focused on playtesting and experimenting that I didn't invest much time on commit messages.

File: mods/hv/weapons/missiles.yaml [hunk 1]
```diff
@@ -295,27 +288,28 @@ TowerMissile:
 DropshipMissile:
```
Comment (Mailaender) [inline mods/hv/weapons/missiles.yaml:288]: You removed `DropshipMissile`. Why is it still here?

File: mods/hv/rules/defaults.yaml [hunk 2]
```diff
@@ -389,7 +389,6 @@
 		WaitDistanceFromResupplyBase: 4c0
 		TakeOffOnResupply: true
 		VTOL: true
-		LandableTerrainTypes: Clear, Ore, Crater, Road, Grass, Grass Pit, Mountain, Snow, Red Snow, Sand, Stone, Tech, Rail, Blocked, Dirt
```
Comment (Mailaender) [inline mods/hv/rules/defaults.yaml:392]: Why are you removing this and duplicating it elsewhere?

File: mods/hv/weapons/missiles.yaml [hunk 3]
```diff
@@ -295,27 +288,28 @@ TowerMissile:
...
 	Inherits: ^AntiGroundMissile
 	Inherits: BigSplash
-	ReloadDelay: 100
-	Range: 8c0
+	ReloadDelay: 70
+	Range: 6c0
+	MinRange: 0c768
```
Comment (Mailaender) [inline mods/hv/weapons/missiles.yaml:293]: And why did you make changes to it?

File: mods/hv/rules/aircraft.yaml [hunk 4]
```diff
@@ -299,16 +310,8 @@ COPTER2:
...
-		TurnSpeed: 80
-		Offset: 600,0,100
-	AttackTurreted:
-		RequiresCondition: airborne
-	Armament:
-		Weapon: Vulcan
-		LocalOffset: 0,0,-400
 	Cargo:
-		Types: Scout
+		Types: Scout, Vehicle
```
Comment (Mailaender) [inline mods/hv/rules/aircraft.yaml:314]: This is reserved for infantry.

File: mods/hv/weapons/missiles.yaml (hunk 3 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/weapons/missiles.yaml:304]: Since Dropships won't have weapons anymore, their weapon actors should be removed.

File: mods/hv/rules/defaults.yaml (hunk 2 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/defaults.yaml:395]: So I don't have to remove it from Elite Pods, Attack Helicopters, Banshees. By default, maybe it should be better to not have 'LandableTerrainTypes' in the base actor, and have this property on actors that should be able to land, but I may be wrong.

File: mods/hv/rules/aircraft.yaml [hunk 5]
```diff
@@ -402,7 +406,6 @@ BALLOON:
 DROPSHIP:
 	Inherits: ^Helicopter
 	Inherits@Smoke: ^SmokeEmitter
-	Inherits@AutoTarget: ^AutoTargetGroundAssaultMove
```
[the remark this answers is not in the document]
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/aircraft.yaml:405]: Even if Dropships do not attack anymore? I'll add it back then!

[the remark this answers is not in the document]
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/aircraft.yaml:405]: It seems it's not breaking. Maybe I shouldn't add it.

File: mods/hv/weapons/missiles.yaml (hunk 1 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/weapons/missiles.yaml:299]: Removed now!

File: mods/hv/rules/aircraft.yaml [hunk 6]
```diff
@@ -411,11 +414,13 @@ DROPSHIP:
 		Queue: Aircraft
 		BuildPaletteOrder: 50
 		Prerequisites: ~starport.yi
-		Description: Vehicle Transport Shuttle.\n  Armed with light missiles.\n  Can load pods\n  and lift one vehicle.
+		Description: Vehicle Transport Shuttle.\n  Armed with light machine gun.\n  Can load pods\n  and lift one vehicle.
```
Comment (Mailaender) [inline mods/hv/rules/aircraft.yaml:417]: Why do you say this has machine guns when you only removed the missiles?

File: mods/hv/rules/aircraft.yaml [hunk 7]
```diff
@@ -272,11 +280,13 @@ COPTER2:
 		Queue: Aircraft
 		BuildPaletteOrder: 50
 		Prerequisites: ~starport.sc
-		Description: Vehicle Transport Helicopter.\n  Armed with light machine gun.\n\n  Can load pods\n  and lift one vehicle.
+		Description: Vehicle Transport Helicopter.\n  Armed with light missiles.\n  Can load pods\n  and lift one vehicle.
```
Comment (Mailaender) [inline mods/hv/rules/aircraft.yaml:283]: You just removed the machine gun and forgot about the missiles.

**Comment by Mailaender** [comment]
Can you give a reason why you removed all weapons from the transport aircraft?

**Comment by WilliamWeberBerrutti** [comment]
> Can you give a reason why you removed all weapons from the transport aircraft?

It's because air transports purpose is to transport units, not attack, so I increased their resistance and made them having Heavy armor type (similarly to what happens with Transport Tank and Water Transport).

When it's added an anti-air air unit, this unit could be good to beat air units, including air transports. I know having a mass of helis and planes

Maybe it could be interesting to let they attack if they are loaded by pods, like it happens with Transport Tanks? Maybe this could be done for Water Transports too?
